.env.example

```diff
@@ -1,2 +1,5 @@
-CLOUDINARY_API_KEY=
+CLOUDINARY_API_KEY=sdslkdjslkdjlksdj
```

Comment:
@Nziranziza  our CLOUDINARY_API_KEY is exposed here! please try to remove it!

Reply:
Ok let me handle it